sdk/src/manifest.rs

```diff
@@ -570,7 +570,7 @@ pub(crate) mod tests {
 
     use crate::{
         assertions::{c2pa_action, Action, Actions},
-        openssl::temp_signer::get_signer,
+        openssl::temp_signer::get_temp_signer,
```

Comment:
Why is it called temp_signer?  There is nothing temp about it if you pass the correct certificates.

Reply:
The temp_signers are the functions used for tests, where you give it a single path and it generates temp signatures in that folder. I'm not sure if we need to export the by-alg version in this case.

The regular get_signer and get_signer_from_files use existing certs.